dashboard/report/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 20px;
+    background-color: #f2f2f2;
+    color: #333;
+}
+
+h1 {
```

Comment:
I'd reconsider customizing such generic types as h, p, ul, li, maybe just add some custom `report` css className (or a few classes since the properties differ)?